api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -449,3 +469,33 @@ message Plan {
   // successfully and data processing is complete.
   repeated TelemetryMetadata telmetry_metadata = 13;
 }
+
+// Request for the 'AddTle' method.
+message AddTleRequest {
+  // The first line of the TLE. Not a title line.
```

Comment:
Yeah let's keep `satellite_id` in here for consistency, clarity, and we can use it as a double-check vs the norad ID. It looks like your comment on `AddTle` is already ready for it too :)